Vagrantfile

```diff
@@ -78,8 +78,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.define("el8", primary: true) do |vm_config|
-    vm_config.vm.box = "centos.cloud/centos8s"
-    vm_config.vm.box_url = "https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-Vagrant-8-latest.x86_64.vagrant-libvirt.box"
+    vm_config.vm.box = "almalinux/8"
+    vm_config.vm.box_url = "https://app.vagrantup.com/almalinux/boxes/8/versions/8.10.20240604/providers/libvirt/amd64/vagrant.box"
```

Comment:
Let's remove the box_url variable here, because it points to a specific version. If we just live it to the box name (almalinux/8), we'll always get the latest.

The only reason we set the box_url for centos images was because the Centos images in vagrant weren't actually updated frequently by the centos devs, so we had to point to `cloud.centos.org`.

Reply:
Updated